_posts/2017-09-25-scastie-embedding.md

```diff
@@ -0,0 +1,142 @@
+---
+category: blog
+by: Guillaume Massé
+title: "Announcing Scastie Embedding"
+date: 25-9-2017
+---
+
+We are excited to announce the availability of a new core feature of Scastie: Embedding.
+
+# What is Scastie?
+
+If you don't know about Scastie yet, here's a recap from our previous announcement.
+
+**Scastie is... Scala + sbt in your browser!** You can:
+
+- use any version of Scala (scalac), or even alternate backends such as Dotty, Scala.js, Scala Native, and Typelevel Scala.
+- use any library published on Maven.
+- save and share Scala programs/builds with anybody
+
+This is all possible because Scastie is based on sbt, allowing us to support newer
+Scala versions and resolve library dependencies on Maven.
+
+Want to see what Scastie can do? Check out this embedding below!
+
+<script src="https://scastie.scala-lang.org/d4Zr5VLvQAirvPwIr2bqMA.js"></script>
+
+## How do I embed Scastie in my website/documentation
+
+Since it's possible to use external libraries, Scastie is an excellant tool to add to your library documentation or your blog post. Let's explore the available methods to bootstrap the embedding mode.
+
+## How do I embed Scastie?
```

Comment:
Do we really need this heading? If yes, why is it at the same level as the previous one?

Reply:
ops